helix-term/src/keymap.rs

```diff
@@ -497,6 +497,7 @@ impl Default for Keymaps {
                 "P" => paste_clipboard_before,
                 "R" => replace_selections_with_clipboard,
                 "space" => keep_primary_selection,
+                "/" => global_search,
```

Comment:
This needs documenting in `book/keymap.md`, I would also mention in a `> NOTE:` that it works well together with `space+'` to resume looking at search results after opening a file.